### PR TITLE
Official Tournament Support Eligibility Changes

### PR DESCRIPTION
**Pull request by DRCallaghan**
## Relevant Changes

- Formally disallow async matches. The Tournament Committee made this change to preserve competitive integrity across badged tournaments.
- Allow eliminated players as refs, playtesters. The Tournament Committee made this change due to lack of concern over either of these roles compromising the competitive integrity of well-run tournaments. In cases where competitive integrity is compromised, hosts of well-run tournaments are expected to resolve the issue or have their support revoked.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/Official_support/fr.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ tags:
   - badges
   - badged
 outdated_translation: true
-outdated_since: d741631f9ac3b2376d85e380e4d1d8a82cca28e5
+outdate_since: 936ee8446406b7b2340514ada4970f608c31908a
```
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/fr.md:7]: previous commit hash needs to be kept

```suggestion
outdate_since: d741631f9ac3b2376d85e380e4d1d8a82cca28e5
```

File: wiki/Tournaments/Official_support/de.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ tags:
   - badged
   - Abzeichen
 outdated_translation: true
-outdated_since: 87e666b9ddf53bdfb7dfdbfe70f2589824f96d37
+outdate_since: 936ee8446406b7b2340514ada4970f608c31908a
```
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/de.md:8]:
```suggestion
outdate_since: 87e666b9ddf53bdfb7dfdbfe70f2589824f96d37
```

File: wiki/Tournaments/Official_support/zh.md [hunk 3]
```diff
@@ -9,6 +9,8 @@ tags:
   - 申牌
   - 比赛
   - 锦标赛
+outdated_translation: true
+outdate_since: 936ee8446406b7b2340514ada4970f608c31908a
```
Comment (LeoFLT) [inline wiki/Tournaments/Official_support/zh.md:13]:
```suggestion
outdated_since: 936ee8446406b7b2340514ada4970f608c31908a
```

File: wiki/Tournaments/Official_support/fr.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ tags:
   - badges
   - badged
 outdated_translation: true
-outdated_since: d741631f9ac3b2376d85e380e4d1d8a82cca28e5
+outdate_since: d741631f9ac3b2376d85e380e4d1d8a82cca28e5
```
Comment (LeoFLT) [inline wiki/Tournaments/Official_support/fr.md:7]:
```suggestion
outdated_since: d741631f9ac3b2376d85e380e4d1d8a82cca28e5
```

File: wiki/Tournaments/Official_support/de.md [hunk 5]
```diff
@@ -5,7 +5,7 @@ tags:
   - badged
   - Abzeichen
 outdated_translation: true
-outdated_since: 87e666b9ddf53bdfb7dfdbfe70f2589824f96d37
+outdate_since: 87e666b9ddf53bdfb7dfdbfe70f2589824f96d37
```
Comment (LeoFLT) [inline wiki/Tournaments/Official_support/de.md:8]:
```suggestion
outdated_since: 87e666b9ddf53bdfb7dfdbfe70f2589824f96d37
```

File: wiki/Tournaments/Official_support/zh.md (hunk 3 above)
Comment (LeoFLT) [inline wiki/Tournaments/Official_support/zh.md:13]:
```suggestion
outdated_since: 936ee8446406b7b2340514ada4970f608c31908a
```

**Review by LeoFLT — CHANGES_REQUESTED** [review]

**Comment by LeoFLT** [comment]
It can be merged if no further issues are found, it was discussed in a tournament committee meeting.

**Review by LeoFLT — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**